modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -42,6 +42,23 @@ CALL (i) {
 This option applies an exponential delay between retries for transaction batches failing due to transient errors, with an optional maximum retry duration, and handles failure based on a specified fallback error mode if the transaction does not succeed within the given time.
 For more information, see xref:subqueries/subqueries-in-transactions#on-error-retry[`CALL` subqueries in transactions -> `ON ERROR RETRY`].
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+GRANT READ {*} ON GRAPH * FOR ()-[r]-() WHERE r.securityLevel > 3 TO regularUsers
+----
+[source, cypher, role=noheader]
+----
+GRANT TRAVERSE ON GRAPH * FOR ()-[r:KNOWS]-() WHERE r.classification IS NULL TO regularUsers
+----
+[source, cypher, role=noheader]
+----
+DENY MATCH {*} ON GRAPH * FOR ()-[r]-() WHERE r.classification <> 'UNCLASSIFIED' TO regularUsers
+----
+| Property-based access control for relationships is now supported in Enterprise Edition. For more information, see link:{neo4j-docs-base-uri}/operations-manual/current/authentication-authorization/property-based-access-control[property-based access control]
```

Comment:
```suggestion
| Property-based access control for relationships is now supported in Enterprise Edition. For more information, see link:{neo4j-docs-base-uri}/operations-manual/current/authentication-authorization/property-based-access-control[property-based access control].
```
and probably also split the two sentences on different lines (makes no difference in the docs but makes it easier to read in code
```suggestion
| Property-based access control for relationships is now supported in Enterprise Edition.
For more information, see link:{neo4j-docs-base-uri}/operations-manual/current/authentication-authorization/property-based-access-control[property-based access control].
```